backend/open_webui/routers/audio.py

```
@@ -7,16 +7,21 @@
+from typing import Optional
 
+from fnmatch import fnmatch
 import aiohttp
 import aiofiles
 import requests
 import mimetypes
+from urllib.parse import quote
 
 from fastapi import (
     Depends,
     FastAPI,
```

Ruff (0.12.2)

`fastapi.FastAPI` imported but unused

Remove unused import: `fastapi.FastAPI`

(F401)